pkg/distributor/distributor_retry.go

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package distributor
+
+import (
+	"flag"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+var (
+	errNonPositiveRetryBase          = errors.New("Retry base duration should be a positive value")
+	errNonPositiveMaxAllowedAttempts = errors.New("MaxAllowedAttempts should be a positive value")
```

Comment:
go errors shouldn't start with a capital letter. See [this guide](https://google.github.io/styleguide/go/decisions.html#error-strings) for example